sellotape/sellotape_dj/main_app/templates/explore.html

```diff
@@ -0,0 +1,149 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="musicStreams" class="collapse row align-items-top justify-content-md-center mb-2" data-parent="#streams">
+            <div class="col-8">
+                <div class="card h-auto p-3 overflow-auto">
+                    <div class="list-group">
+                    {% if not music_streams %}
+                        No Music streams!
+                    {% else %}
+                        <h1 style="display-4">Music streams</h1>
+                    {% endif %}
+                    {% for stream in music_streams %}
+                      <a href="{{ stream.link }}" class="list-group-item list-group-item-action flex-column align-items-start">
+                        <div class="d-flex w-100 justify-content-between">
+                          <h5 class="mb-1">{{ stream.title }}</h5>
+                          <small>Aired on {{ stream.airs_on|date }}</small>
+                        </div>
+                        <p class="mb-1"></p>
+                        <small>(Click here to view the stream)</small>
+                      </a>
+                    {% endfor %}
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="blogStreams" class="collapse row align-items-top justify-content-md-center mb-2" data-parent="#streams">
+            <div class="col-8">
+                <div class="card h-auto p-3 overflow-auto">
+                    <div class="list-group">
+                    {% if not blog_streams %}
+                        No Blog streams!
+                    {% else %}
+                        <h1 style="display-4">Blog streams</h1>
+                    {% endif %}
+                    {% for stream in blog_streams %}
+                      <a href="{{ stream.link }}" class="list-group-item list-group-item-action flex-column align-items-start">
+                        <div class="d-flex w-100 justify-content-between">
+                          <h5 class="mb-1">{{ stream.title }}</h5>
+                          <small>Aired on {{ stream.airs_on|date }}</small>
+                        </div>
+                        <p class="mb-1"></p>
+                        <small>(Click here to view the stream)</small>
+                      </a>
+                    {% endfor %}
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="topRated" class="collapse row align-items-top justify-content-md-center mb-2" data-parent="#streams">
+            <div class="col-8">
+                <div class="card h-auto p-3 overflow-auto">
+                    <div class="list-group">
+                    {% if not top_rated %}
+                        No Users!
+                    {% else %}
+                        <h1 style="display-4">Top Rated Users</h1>
+                    {% endif %}
+                    {% for top in top_rated %}

```

Comment:
What is `top`? Is that a user profile? Please use meaningful names - it will help the readability! :)